src/canvas-player-controls.js

```diff
@@ -71,19 +71,37 @@ class CanvasPlayerControls extends videojs.EventTarget {
   }
 
   onMoveEnd(e) {
+
+    // We want to have the same behavior in VR360 Player and standar player.
+    // in touchend we want to know if was a touch click, for a click we show the bar,
+    // otherwise continue with the mouse logic.
+    if (e.type === 'touchend' && this.touchMoveCount_ < 7) {
```

Comment:
why 7?

Reply:
Commonly when you touch the screen on and android device looking for "touch for a click" not "move touch" you get between 1 - 6 `touchmove`

Reply:
Looks like Video.js uses 10 as the threshold, might be better to just use that: https://github.com/videojs/video.js/blob/master/src/js/component.js#L1147-L1150.